dags/merino_jobs.py

```diff
@@ -89,7 +90,30 @@ def merino_job(
             "--total-docs",
             "6600000",  # Estimate of the total number of documents in wikipedia index
             "--elasticsearch-url",
-            str(conn.host),
+            str(staging_connection.host),
+            "--gcs-path",
+            "moz-fx-data-prod-external-data/contextual-services/merino-jobs/wikipedia-exports",
+            "--gcp-project",
+            "moz-fx-data-shared-nonprod",
```

Comment:
sorry i missed this originally. this refers to what gcp project the file was downloaded to. it's should be the same across all tasks eg `moz-fx-data-shared-prod`